source/community/puppet_projects_workflow.markdown

```diff
@@ -182,7 +182,61 @@ Please either (a) update it with the information I've requested and re-assign it
 
 #### Code Insufficient
 
-I've put this ticket's status into "Failed Review" and assigned it to you. Please either (a) update the ticket or pull request with a version of code which addresses the issues, or (b) assign it back to me if you are blocked on the problem and I'll move things forward.
+I’ve put this ticket’s status into “Failed Review” and assigned it to you. Please either (a) update the ticket or pull request with a version of code which addresses the issues, or (b) assign it back to me if you are blocked on the problem and I’ll move things forward.
```

Comment:
I don't think we still have the "Failed Review" status; the workflow should be to set it back to Ready for Eng